src/themes/clean/templates/core/news/index.html

```diff
@@ -12,10 +12,10 @@ <h1>{{ journal_settings.news.news_title }}</h1>
         </div>
         {% for item in news_items %}
             <div class="col-md-12">
-                <h2>{{ item.title|safe }}</h2>
+                <h2 id="news-item-{{ item.pk }}">{{ item.title|safe }}</h2>
                 <p class="byline">{{ item.byline }} on {{ item.posted|date_human }}</p>
                 <p>{{ item.body|truncatewords_html:50|safe }}</p>
-                <a href="{% url 'core_news_item' item.pk %}">{% trans 'Read More' %}</a>
+                <a href="{% url 'core_news_item' item.pk %}" aria-describedby="news-item-{{ item.pk }}">{% trans 'Read more' %}</a>
```

Comment:
If changing the key to this translation target we need to:
 - Update the key in the translation files
 - Update the translations so they match the default case